ui/app/com/gu/recipeasy/views/app.scala.html

```diff
@@ -8,18 +13,35 @@
     }
     </style>
     <div class="wrapper">
-      <img class="large-image" src="@routes.Assets.versioned("images/main.jpg")" alt="" />
+        <img class="large-image" src="@routes.Assets.versioned("images/main.jpg")" alt="" />
         <div class="title-container">
             <div class="container">
                 <img class="img-fluid" src="@routes.Assets.versioned("images/logo.png")" alt="" />
                 <div class="title-line--first">Our one off tool</div>
                 <div class="title-line--second">for structuring recipes</div>
-            <div class="text-xs-center" id="curation-caption">
-                Stage one (structuring) completion @{progressBarPercentage}%
+                <div class="text-xs-center" id="curation-caption">
+                    Stage one (structuring) completion @{progressBarPercentage}%
+                </div>
+                <progress class="progress" value="@{progressBarPercentage}" max="100" aria-describedby="curation-caption"></progress>
+            </div>
+        </div>
+        <div class="container" style="padding:30px 0 20px 0;background-color: rgba(255, 255, 255, 0.6);">
+            <div class="col-md-6">
+                <h1>Hi @{userFirstname}, your stats</h1>
+                <span class=""><strong>Contributions:</strong> </span><span class="">@{userStats(PStatsCurationCount)} curated, @{userStats(PStatsVerificationCount)} verified, @{userStats(PStatsFinalisationCount)} finalised, @{yourParticipation(generalStats, userStats)}</span><br />
```

Comment:
Let's split out percentage contributed on to a new line
`Percentage contribution: 0.04%`